README.md

```diff
@@ -2150,7 +2188,45 @@
 ## Organization
 
 <a name="ts-modules"></a>
-  - [28.1](#28.1) <a name='28.1'></a> Organization/module placeholder.
+  - [28.1](#28.1) <a name='28.1'></a> 1 file per logical component, and each file should be divided into logical divisions via modules. 
+
+  ```javascript
+  module Automobile {
+
+    module Honda {
+
+    }
+
+  }
+  ```
+  
+  - [28.2](#28.2) <a name='28.2'></a> Export one main module per file so it can be required by other files.
+
+  ```javascript
+  module Automobile {
+
+    // hidden module, will not be accessible via "require"
+    Honda {
+
+    }
+  
+    // public module, will be accessible via "require"
+    export Ford {
+
+      export function vroom() {
+
+        console.log('vroom!');
+
+      }
+
+    }
+
+  }
+
+  export default Automobile;
+  ```
+
+- [28.3](#28.3) <a name='28.3'></a> Modules should be Pascal case.
```

Comment:
This is covered in the naming conventions section: https://github.com/jacobbridges/typescript/blob/master/README.md#22.2
You could add an example for modules in this section.